src/chrome/playSound.js

```diff
@@ -0,0 +1,20 @@
+import { SOUND_URL, SOUND_REPEAT } from '../constants';
+
+// Get sound options and play if enabled
+
+export const getAndPlaySoundNotificationOption = (store, mainAudioPlayer) => {
```

Comment:
Also, since `mainAudioPlayer` isn't used anywhere else, I think it can just be a top-level constant in the module instead of being passed in. Also it can just be called `audioPlayer` probably.